automatic/brave/tools/install.ahk

```diff
@@ -0,0 +1,9 @@
+#NoEnv
```

Comment:
Why do we need AHK script at all ?`

Reply:
Despite "Silent" in the installer's name, the browser automatically starts after being installed.

Reply:
When you install chrome via chocolatey it also shows the installer.

Reply:
@JipSterk Which is completely unrelated to this PR. Please keep comments on topic.

Reply:
> Despite "Silent" in the installer's name, the browser automatically starts after being installed.

Because its about **silent installation**, not sillent running.
You should kill a process in this case, with timeout.

Reply:
@majkinetor do mean without the AHK script?

Reply:
As far as I understand, it can't be done for this package, since Brave starts before the installation process completed. I misspoke, sorry.

Good news: the new beta version is completely silent. I means the next stable version will be silent too.